### DISCUSSION TITLE
Docker, can't train

### DISCUSSION THREAD

**Discussion by ankhafizov** [open-mmlab/mmaction2#2730] [General]
Are there any tested configuration for modern gpu (for example nvidia a40). Can't use recommended torch 1.8.1 because it is out of date for my device, but can't use 1.13.1 because it is not compatible with the code either and raises CUDA ERROR. Which torch to choose to run train in docker? By the way, my setup has 4 nvidia a40, each with 40 Gb of memory

**Comment by ankhafizov** [comment]
finally it worked with 
ARG PYTORCH="1.8.1"
ARG CUDA="10.2"
ARG CUDNN="7"

after a bit of wait

**Comment by zolali** [comment]
what is version of docker in buffer overflow server lab?and how I run docker-compose.yml file?when I write the command" "
